manuscripts/report.py

```diff
@@ -605,20 +606,31 @@ def create_data_figs(self):
         logger.info("Data and figs done")
 
     @classmethod
-    def period_name(cls, pdate, interval='quarter', offset=None):
+    def build_period_name(cls, pdate, interval='quarter', offset=None, start_date=False):
+        """ If not start_date, pdate is the end of the period date """
```

Comment:
I think it would be better that the first line of the function comment says what the function does. Then, this line could be an additional explanation, and after that I'm missing the parameters.

Reply:
Great! I will follow this advice. Let's start commenting right the code in manuscripts.